setup.py

```diff
@@ -87,12 +86,12 @@ def run(self):
         self._create_env_script()
         _install_data.run(self)
 
-setup(name = 'python-maxwidgets',
-      version = '0.9.6',
-      description = 'A collection of reusable Taurus widgets',
-      packages = packages,
-      package_dir = {'maxwidgets' : 'src'},
-      data_files = [('/etc/profile.d', [env_script])],
-      cmdclass = {#'build_py' : build_py,
-                  'install_data' : install_data},
-     )
+
+setup(name='python3-maxwidgets',
+      version='1.0.0',
+      description='A collection of reusable Taurus widgets',
+      packages=packages,
+      package_dir={'maxwidgets': 'src'},
+      data_files=[('/etc/profile.d', [env_script])],
```

Comment:
I really don't think a python package should try to install something under `/etc/profile.d`.
This isn't gonna work if you install the package in a virtualenv.

It only works when using a RPM. That should be done in the RPM, not python package.

Plus I don't think setting `TAURUSQTDESIGNERPATH` is the proper way to add widgets to taurus today.
See https://www.taurus-scada.org/devel/plugins.html and https://gitlab.com/taurus-org/taurus_pyqtgraph as an example.